### ISSUE TITLE
Arduino receiving UDP or IP

### ISSUE THREAD

**Issue by jaapnoordzij** [pschatzmann/arduino-midi#7]
I want to use the possibility to receive Midi through UDP (or IP).
In your Desktop examples you provide for both the possibility:
- Sending MIDI to the Arduino TCP Server
- Sending MIDI to the Arduino UDP Server

Both state: "If we have deployed the MidiUDPServer sketch we can send MIDI messages to Arduino:"

But I cannot find an MidiUDPServer sketch.
- There is only a "udp-send" example sketch but no udp-receive (with UDP server)
- There is an ip-receive example but the code in the loop is using "out" (sending messages)

Could you please give me some hints to get this working ? 
Currently I am using your AppleMidi solution which works fine.

Thank you
Jaap

**Comment by pschatzmann** [comment]
I think you can use the applemidi-receive.ino as starting point and just replace the AppleMidiServer with the MidiUdpServer

**Comment by pschatzmann** [comment]
Did you call the [begin with the correct port](https://pschatzmann.github.io/arduino-midi/html/classmidi_1_1_midi_udp_server.html). If you dont specify anything the default port 5008 is used.
Can you share your sketch ?

**Comment by jaapnoordzij** [comment]
```
/**
 * @file applemidi-receive.ino
 * @author Phil Schatzmann
 * @brief Receiving Midi messages using Apple Midi
 * @date 2021-12-24
 * 
 * @copyright Copyright (c) 2021
 */

#include <WiFi.h>
#include "Midi.h"

MidiCallbackAction action;
MidiUdpServer apple(&action);

const char *SSID = "xxx";
const char *PWD = "yyy";

void onNoteOn(uint8_t channel, uint8_t note, uint8_t velocity) {
  Serial.print("onNoteOn: ");
  Serial.println(note);
}

void onNoteOff(uint8_t channel, uint8_t note, uint8_t velocity) {
  Serial.print("onNoteOff: ");
  Serial.println(note);
}

void setupWIFI() {
  Serial.begin(119200);
  MidiLogLevel = MidiInfo;

  WiFi.begin(SSID, PWD);
  while (WiFi.status() != WL_CONNECTED) {
    Serial.print('.');
    delay(1000);
  }

  Serial.print("Connected to IP address: ");
  Serial.println(WiFi.localIP());
}

void setup() {
  action.setCallbacks(onNoteOn, onNoteOff);
  setupWIFI();
  apple.begin(5004);
}

void loop() {
  // if (!apple.loop()){
  //   delay(10);
  // }
  apple.loop();
  // delay(10);
}```

**Comment by pschatzmann** [comment]
It think it should be something like apple.begin(IPAddress(192, 168, 2, 1),5004); where the address would point back to the sender

**Comment by jaapnoordzij** [comment]
You are right, the MidiUdpServer begin requires an IP address, wonder why the compiler does not complain.
However, No change, still no incoming messages
`  apple.begin(IPAddress(192,168,2,111),5004);` which is the address of the sender, my Mac
